whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/integration/SignedMessageIntegrationSpec.groovy

```diff
@@ -3920,4 +3920,70 @@ class SignedMessageIntegrationSpec extends BaseWhoisSourceSpec {
       ack =~ "Create SUCCEEDED: \\[person\\] FP1-TEST   First Person"
   }
 
+  @Ignore("TODO: [ES] We do not support base64 encoded content yet")
```

Comment:
Suggestion: Maybe assert this currently fails, so the test passes? so when this is implemented, then the test will fail, which will then be fixed. 

This prevents a situation, whereby, if the fix is implemented by another developer apart from you, a duplicated test case can be added, since there is a chance they did not know this ignored test existed.

Reply:
I prefer to `@Ignore` the failing test, then remove the annotation when implementing, then the test itself doesn't have to be changed, it will pass once the change is implemented.

Hopefully we'll get to resolve this in the short term (there's an existing Jira issue to track it).